h/templates/help.html.jinja2

```diff
@@ -54,20 +54,20 @@
           with the application:</p>
         <div class="feature-content">
           <section class="feature">
-            <h3 class="feature-heading"><i class="feature-icon h-icon-annotate"></i> Notes</h3>
+            <h3 class="feature-heading"><i class="feature-icon h-icon-annotate"></i> Annotate</h3>
```

Comment:
The sub-headings here are nouns rather than verbs because the parent heading is "Annotation Types". Verbs would be appropriate if the parent heading was something like "Ways to annotate". "Annotations" would be better than "Notes" here though, since this section is talking about annotations that are associated with a specific part of the document as opposed to annotations about the whole document, which we call "Page Notes".